src/pyproject_hooks/_impl.py

```diff
@@ -104,12 +130,12 @@ class BuildBackendHookCaller:
 
     def __init__(
         self,
-        source_dir,
-        build_backend,
-        backend_path=None,
-        runner=None,
-        python_executable=None,
-    ):
+        source_dir: str,
+        build_backend: str,
+        backend_path: Optional[List[str]] = None,
+        runner: Optional["SubprocessRunner"] = None,
```

Comment:
I don't think the quotes here are necessary:

```suggestion
        runner: Optional[SubprocessRunner] = None,
```

Reply:
It isn't for 3.8+, but we're preserving 3.7 compatibility (see below).